src/cmd-buildextend-live

```diff
@@ -341,6 +342,13 @@ def generate_iso():
                 buf = newbuf
                 karg_area_end = re.search(r'(#+) COREOS_KARG_EMBED_AREA\n', buf)
                 if karg_area_end is not None:
+                    if len(default_kargs) == 0:
+                        default_kargs = buf[karg_area_start.start():karg_area_end.end()]
```

Comment:
Shouldn't this be

```
default_kargs = buf[karg_area_start.start():karg_area_end.start()]
```

?

Reply:
In this case length will be less than for other karg files. I though, that it's better to have similar files (ending with `COREOS_KARG_EMBED_AREA`), than handing different lengths in `coreos-installer`. 
Or, we can modify `header` format and store there `TLV` - like structure for each karg file, where `Type` can be `ro` or `rw`